distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestMessageBroker.java

```diff
@@ -105,46 +98,14 @@
 
     private static CamelContext camelContext;
 
-    protected final AbstractIntegrationTest.DynamicPort amqpPort =
-            new AbstractIntegrationTest.DynamicPort(6);
-
-    protected final AbstractIntegrationTest.DynamicPort artemisPort =
-            new AbstractIntegrationTest.DynamicPort(7);
-
-    protected final AbstractIntegrationTest.DynamicPort openwirePort =
-            new AbstractIntegrationTest.DynamicPort(8);
-
     private String messageId;
 
-    @Override
-    protected Option[] configureConfigurationPorts() throws URISyntaxException, IOException {
-        return combineOptions(super.configureConfigurationPorts(),
-                new Option[] {editConfigurationFilePut("etc/system.properties",
-                        "artemis.amqp.port",
-                        amqpPort.getPort()), editConfigurationFilePut("etc/system.properties",
-                        "artemis.multiprotocol.port",
-                        artemisPort.getPort()), editConfigurationFilePut("etc/system.properties",
-                        "artemis.openwire.port",
-                        openwirePort.getPort())});
-    }
-
-    @Override
-    protected Option[] configureLogLevel() {
-        return combineOptions(super.configureLogLevel(),
-                options(editConfigurationFilePut("etc/" + LOG_CONFIG_PID + ".cfg",
-                        "log4j.additivity.org.apache.activemq.artemis",
-                        "true")));
-    }
-
     @BeforeExam
     public void beforeExam() throws Exception {
-        basePort = getBasePort();
-        getAdminConfig().setLogLevels();
+        waitForSystemReady();
 
-        getServiceManager().waitForRequiredApps(REQUIRED_APPS);
+        getServiceManager().waitForRequiredApps("broker-app");
```

Comment:
@ryeats Since the Artemis system properties are only used by `broker-app`, would it work if those `DynamicPort`s were defined in this test class and `System.setProperty()` was called before starting `broker-app`?